spec/std/http/cookie_spec.cr

```diff
@@ -146,14 +146,36 @@ module HTTP
 
       it "parse max-age as seconds from Time.now" do
         cookie = parse_set_cookie("a=1; max-age=10")
-        delta = cookie.expires.not_nil! - Time.now
-        delta.should be > 9.seconds
-        delta.should be < 11.seconds
+        delta = cookie.max_age
+        delta.should eq 10.seconds
```

Comment:
you could skip `delta` intermediary variable altogether since there's one call only.